config/samples/k8up_v1alpha1_schedule.yaml

```diff
@@ -52,7 +52,7 @@ spec:
       requests:
         memory: "64Mi"
         cpu: "250m"
-    schedule: '*/5 * * * *'
+    schedule: '@hourly-random'
```

Comment:
The problem is, that `@hourly` is an allowed syntax which results in `0 * * * *`
https://crontab.guru/#@hourly

Reply:
1. As discussed in https://github.com/vshn/k8up/issues/116 and also with @Kidswiss, we want to keep the `-random` suffixes. Should we later implement another method to generate schedules, this would allow us to add more different kind of implementations, e.g. `@hourly-smart`, `@daily-nightly` etc.
2. As @srueg mentioned correctly, we want to keep the possibility to set the pre-defined schedules